packages/headless/src/controllers/smart-snippet/headless-smart-snippet.ts

```diff
@@ -125,6 +155,22 @@ export function buildSmartSnippet(engine: SearchEngine): SmartSnippet {
       engine.dispatch(logDislikeSmartSnippet());
       engine.dispatch(dislikeSmartSnippet());
     },
+    openFeedbackModal() {
```

Comment:
What's the difference between this and the `like`/`dislike` state?

If we remove the state, do we keep this method for the purpose of logging analytics, or just export the actions?

Reply:
Good point about the like/dislike...

I went into the old design doc for smart snippet, and here's what I found.

We added it to encourage/"force" people to do it to log analytics so we get some raw material for ML. So that users read the controller API and think about implementing it in their own UI.

So with that aspect in mind, then I guess it makes sense to have the same logic for the feedback mechanism... Even if it's purely UI. 

So, I take back my comment then, and we can leave it in Headless. It's good that it's there if it helps encourage best practices.